REFERENCE.md

```diff
@@ -970,15 +970,15 @@ Default value: `present`
 
 ##### <a name="-apt--source--release"></a>`release`
 
-Data type: `Optional[String]`
+Data type: `Optional[Pattern[/\A[[:alnum:]\s-]+\z/]]`
```

Comment:
That's arguably less readable than `String`, but for now this is probably fine?  In the long term, maybe we can add "wrapper" type aliases in stdlib like `Identifier` (`/\A[a-z][a-z0-9]*\z/i`).